CHANGELOG.md

```diff
@@ -6,7 +6,10 @@
 
 ### Fixed
 * <How do the end-user experience this issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
-* None.
+* Fixed a change of mode from Strong to All when removing links from an embedded object that links to a tombstone. This affects sync apps that use embedded objects which have a `Lst<Mixed>` that contains a link to another top level object which has been deleted by another sync client (creating a tombstone locally). In this particular case, the switch would cause any remaining link removals to recursively delete the destination object if there were no other links to it. ([#7828](https://github.com/realm/realm-core/issues/7828), since 14.0.0-beta.0)
+* Fixed `Table::remove_object_recursive` which wouldn't recursively follow links through a single `Mixed` property. ([#7829](https://github.com/realm/realm-core/issues/7829), likely since the introduction of Mixed)
```

Comment:
This maybe goes under Internal since we don't actually expose remove_object_recursive().

Reply:
Valid point. It is part of core's public API, so I wasn't sure if this feature was exposed by any SDK or not.